src/main/java/uk/gov/hmcts/divorce/citizen/notification/ApplicationSubmittedNotification.java

```diff
@@ -0,0 +1,47 @@
+package uk.gov.hmcts.divorce.citizen.notification;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.divorce.common.model.CaseData;
+import uk.gov.hmcts.divorce.notification.NotificationService;
+
+import java.time.format.DateTimeFormatter;
+import java.util.HashMap;
+import java.util.Map;
+
+import static java.lang.String.join;
+import static java.time.format.DateTimeFormatter.ofPattern;
+import static uk.gov.hmcts.divorce.notification.EmailTemplateName.APPLICATION_SUBMITTED;
+import static uk.gov.hmcts.divorce.notification.NotificationConstants.APPLICATION_REFERENCE;
+import static uk.gov.hmcts.divorce.notification.NotificationConstants.SUBMISSION_RESPONSE_DATE;
+
+@Component
+public class ApplicationSubmittedNotification {
+
+    private static final DateTimeFormatter formatter = ofPattern("d MMMM yyyy");
+
+    @Autowired
+    private NotificationService notificationService;
+
+    @Autowired
+    private CommonContent commonContent;
+
+    public void send(CaseData caseData, Long id) {
+        Map<String, String> templateVars = new HashMap<>();
```

Comment:
some logging would be nice here like which notification is sent for which case id